model/task/task.go

```diff
@@ -1594,7 +1594,13 @@ func (t *Task) GetDisplayTask() (*Task, error) {
 	if t.DisplayTask != nil {
 		return t.DisplayTask, nil
 	}
-	dt, err := FindOne(ByExecutionTask(t.Id))
+	var dt *Task
+	var err error
+	if t.Archived {
+		dt, err = FindOneOld(ByExecutionTask(t.Id))
```

Comment:
what is this coming to address?

Reply:
There's a bug in this function where it doesn't return the parent display task if the task is in the old_tasks collection. I don't need it in the new approach, but I'm going to fix it anyway

Reply:
do we need to convert back from the old_task id to find its display task? (since the execution task ids in the display task are not updated when the display task is archived)

Reply:
yep, fixed